camayoc/tests/qcs/cli/test_sources.py

```diff
@@ -682,7 +671,7 @@ def test_clear(isolated_filesystem, qpc_server_config, source_type):
         'qpc source clear --name={}'.format(name)
     )
     assert qpc_source_clear.expect(
-        'Source "{}" was not found'.format(name)
+        'Source "{}" was not found.'.format(name)
```

Comment:
The `.` was added and make the test fail. Better to be specific and catch even a character change, don't you think?

Reply:
Hmm, I'm indifferent. I don't think the messages will change that often, @jlprevatt just made a docs PR.